crates/rethnet_eth/src/transaction.rs

```diff
@@ -62,6 +62,49 @@ pub struct EthTransactionRequest {
     pub transaction_type: Option<U256>,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Default)]
```

Comment:
Make sure to provide attribution

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/3383/commits/501f2ca34508b0842e58a96c762c6dd1ce0f94f5